draft-ietf-doh-dns-over-https-latest.mkd

```diff
@@ -520,6 +528,83 @@ Author:  Paul Hoffman, paul.hoffman@icann.org
 ~~~~~
 
+# Privacy Considerations {#PrivacyConsiderations}
+
+{{RFC7626}} discusses DNS Privacy Considerations in both "On the
+wire" (Section 2.4), and "In the server" (Section 2.5) contexts. This
+is also a useful framing for DoH's privacy considerations.
+
+## On The Wire {#OnTheWire}
+
+DoH encrypts DNS traffic and requires authentication of the
+server. This mitigates both passive surveillance {{RFC7258}} and
+active attacks attempting to divert DNS traffic to rogue servers
+({{RFC7626}} Section 2.5.1). DNS over TLS {{RFC7858}} provides
+similar protections, while direct UDP and TCP based transports are
+vulnerable to this class of information leak.
+
+Additionally, the use of the HTTPS default port 443 and the ability to
+mix DoH traffic with other HTTPS traffic on the same connection can
+deter on-path devices from interfering with DNS operations and make
+DNS traffic analysis more difficult.
+
+## In The Server {#InTheServer}
+
+A DoH implementation is built on IP, TCP, TLS, and HTTP. Each layer
+contains one or more common features that can be used to correlate
+different queries to the same identity. DNS transports will generally
+carry the same privacy properties of the layers used to implement
+them. For example, the properties of IP, TCP, and TLS apply to DNS
+over TLS implementations.
+
+The privacy considerations of using the HTTPS layer in DoH are
+incremental to those of DNS over TLS. DoH is not known to introduce
+new concerns beyond those associated with HTTPS.
+
+At the IP level, the client address provides obvious correlation
+information. This can be mitigated by use of a NAT, proxy, VPN, or
+simple address rotation over time. It may be aggravated by use of a
+DNS server that can correlate real-time addressing information with
+other personal identifiers, such as when a DNS server and DHCP server
+are operated by the same entity.
+
+TCP-based solutions may seek performance through the use of TCP Fast
+Open {{RFC7413}}. The cookies used in TCP Fast Open allow servers to
+correlate different TCP sessions together.
+
+TLS based implementations often achieve better handshake performance
+through the use of some form of session resumption mechanism such as
+session tickets {{RFC5077}}. Session resumption creates trivial
+mechanisms for a server to correlate different TLS connections
+together.
+
+HTTP's feature set can also be used for identification and tracking in
+a number of different ways. For example, authentication request header
+fields explicitly identify profiles in use, and HTTP Cookies are
+designed as an explicit state tracking mechanism between the client
+and serving site and often are used as an authentication mechanism.
+
+Additionally, the User-Agent and Accept-Language request header fields
+often convey specific information about the client version or locale.
+This facilitates content negotiation and operational work-arounds for
+implementation bugs. Request header fields that control caching
```

Comment:
Do you want to discuss whether there is a use for these in the context of DoH

Reply:
I don't think they generally match the general advice.. I was convinced cookies are worth a special call out just because of how flexible and ubiquitous they are (but they really match the general advice too.)